examples/examples/z_put.rs

```diff
@@ -27,7 +31,18 @@ async fn main() {
 
     println!("Putting Data ('{key_expr}': '{payload}')...");
     // Refer to z_bytes.rs to see how to serialize different types of message
-    session.put(&key_expr, payload).await.unwrap();
+    for i in 0..usize::MAX {
```

Comment:
The `z_put` example should remain "1 shot".
(Btw, the `if i % 2` thing is an interesting test but has nothing to do in an example imho)

Reply:
It's an unwanted committed file... my bad.